wiki/osu!supporter/en.md

```diff
@@ -1,6 +1,3 @@
----
-outdated: true
```

Comment:
you can't remove the outdated marker if you don't address the issues mentioned in TODO, and I don't feel that what you've changed so far is enough for that

Reply:
should I add the marker back or else what should I add to the page?

Reply:
I've just skimmed through the article again and didn't find many things to improve. I think that for now you can replace `outdated: true` with `needs_cleanup: true`. 

@cl8n blame says the TODO is yours — any details on cleanup ideas other than replacing the images?

Reply:
I'll change that as soon as I'm at home

Reply:
reading over the article some sentences sound odd due to mixing formal and casual style, and the author sounds uncertain about details, which is really strange for a wiki article. I imagine all I meant by my TODO comment was that it needed a quick pass by a native speaker. if these are too nitpicky of reasons for `needs_cleanup` and/or my comment, feel free to remove

Reply:
they aren't, thanks for clarification